specs/sql/t1.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE t1(
```

Comment:
as this tables a specific to this benchmark, I'd prefer to just include these statements inside the setup as I think the y will never be used by any other benchmark.